src/KlaviyoAPI.php

```diff
@@ -176,13 +177,11 @@ private function handleResponse( $response, $statusCode, $isPublic )
     {
         if ( $statusCode == 403 ) {
             throw new KlaviyoAuthenticationException(self::ERROR_INVALID_API_KEY, $statusCode);
-        } else if ( $statusCode == 404 ) {
-            throw new KlaviyoResourceNotFoundException( self::ERROR_RESOURCE_DOES_NOT_EXIST, $statusCode);
```

Comment:
Removing this entirely so we can display the specific response for a 404 from the data-privacy endpoint.